git.go

```diff
@@ -98,9 +97,14 @@ func (c *nativeGitWrapper) Log(options *LogOption) (*Result, error) {
 	jsonStr = jsonStr[:len(jsonStr)-1]
 	jsonStr = fmt.Sprintf(`[%s]`, jsonStr)
 
-	var commits []*GitCommit
-	err = json.Unmarshal([]byte(jsonStr), &commits)
-	if err != nil {
+	var p = &Parser{
+		JsonStr: "",
+		Head:    0,
+		Len:     0,
+	}
+	commits, err := p.Unmarshal([]byte(jsonStr))
+	if nil != err {
+		fmt.Println(err)
```

Comment:
Hi @andy-y-li , can you add a proper error message here instead of only logging the error?